frontend/components/GameVisibilityList.tsx

```diff
@@ -10,35 +11,28 @@ const GamesVisibilityList: React.FC<GameVisibilityListProps> = (props: GameVisib
 
 	return (
 		<>
-			{Array.from(gamesVisibility.values()).map(game=> {
-				return <GameVisibilityToggle key={game.id} name={game.name} isVisible={game.isVisible} id={game.id} />;
+			{Array.from(gamesVisibility.values()).map(entry=> {
```

Comment:
If you want to use the `Map()` of gameId -> Game here I'd suggest to extract `Array.from(gamesVisibility.values())` to a variable first, to make this line easier to parse, eg:

```
const visibilityData  = Array.from(gameVisibility.values());
return (
    <>
        {visibilityData.map(...
```

This will also better expose that the only think you ever do with `gameVisibility` is just get its values, so you don't really need a `Map()` for it. And since you own the creation of that data structure (it's made in `mapStateToProps`) you can just change it to be an array.

What I am saying is, there is a clear benefit of replacing this map with an array from the get go, [see that comment](https://github.com/EvidentlyCube/metroidvania-compendium/pull/10/files/04b7d2265aa4f84eb6489ea04dcb4ad6adcff322..78b86302812f7e5aedcbf793eb377881e3944f4b#r389992846) :).